packages/firebase_inappmessaging/test/firebase_inappmessaging_test.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:firebase_inappmessaging/firebase_inappmessaging.dart';
+
+void main() {
+  const MethodChannel channel = MethodChannel('firebase_inappmessaging');
+
+  setUp(() {
+    channel.setMockMethodCallHandler((MethodCall methodCall) async {
```

Comment:
Can you please add unit tests following the pattern of other plugins?

Reply:
done - let me know what you think!